dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -9,6 +9,15 @@ import '../framework/adb.dart';
 import '../framework/framework.dart';
 import '../framework/utils.dart';
 
+
+TaskFunction createPlatformServiceDriverTest() {
+  return new PerfTest(
```

Comment:
Could you please create a copy of `PerfTest` called `DriverTest` and remove the performance metrics? We don't want to publish metrics that we do not set goals and baselines for.